docs/linux/helper-function/bpf_get_task_stack.md

```diff
@@ -14,7 +14,7 @@ description: "This page documents the 'bpf_get_task_stack' eBPF helper function,
 
 
 <!-- [HELPER_FUNC_DEF] -->
```

Comment:
These bits of the docs between the `<!-- [HELPER_FUNC_DEF] -->` and `<!-- [/HELPER_FUNC_DEF] -->` markers are auto generated from libbpf header file comments.

I agree with taking over ownership and improving them, but we should remove these marker comments, otherwise your changes will be undone next time `make generate-docs` is called